src/bundle/Command/MigrateLegacyMatrixCommand.php

```diff
@@ -81,7 +81,7 @@ public function execute(InputInterface $input, OutputInterface $output)
             if ($answer !== self::CONFIRMATION_ANSWER) {
                 $io->comment('Canceled.');
 
-                exit();
+                return 0;
```

Comment:
```suggestion
                return 1;
```

Canceled action shouldn't return `0` exit code (as it will execute potential following shell commands anyway).

Reply:
@Nattfarinn please check https://github.com/ezsystems/ezplatform-matrix-fieldtype/pull/36/commits/3006656a9d12be7c197e33032678359a674a365c